### PR TITLE
issue #87 cupsfilter showing particular errors.

### PR DESCRIPTION
**Pull request by surajkulriya**
[issue 87](https://github.com/OpenPrinting/cups/issues/87)

### CODE REVIEW

File: scheduler/cupsfilter.c [hunk 1]
```diff
@@ -221,8 +227,10 @@ main(int  argc,				/* I - Number of command-line args */
 		i ++;
 		if (i < argc && !infile)
 		  infile = argv[i];
-		else
-		  usage(opt);
+		else{
```
Comment (michaelrsweet) [inline scheduler/cupsfilter.c:230]: Coding style: CUPS puts open braces on their own line (not K&R), uses (traditional) 8 column tabs, and two spaces of indentation, e.g.:

```
else
{
  _cupsLanfPrintf(stderr, _("%s: Error - ..."), argv[0]);
  usage(NULL);
}
```

File: scheduler/cupsfilter.c [hunk 2]
```diff
@@ -184,9 +184,11 @@ main(int  argc,				/* I - Number of command-line args */
 		i ++;
 		if (i < argc)
 		  num_options = cupsParseOptions(argv[i], num_options, &options);
-		else
+		else{
+      _cupsLangPrintf(stderr,"%s: Error - expected option after \"-a\" option.",argv[0]);
```
Comment (michaelrsweet) [inline scheduler/cupsfilter.c:188]: Messages need to use the `_("string")` macro in order to get scanned by gettext (which we use to build the message catalog).

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
I generally approve of the changes. See my comments about coding style/indentation and the use of the "_" macro for localization.

File: scheduler/cupsfilter.c [hunk 3]
```diff
@@ -235,7 +247,10 @@ main(int  argc,				/* I - Number of command-line args */
 		  srctype = argv[i];
 		}
 		else
-		  usage(opt);
+    {
+      _cupsLangPrintf(stderr,_("%s: Error - expected source MIME type after \"-i\" option."),argv[0]);
+		  usage(NULL);
+    }
```
Comment (surajkulriya) [inline scheduler/cupsfilter.c:253]: I don't exactly know why its showing like this in github. I am using vscode editor for editing files. `else` statement and opening brace `{` are being shown in vertical line there.

**Comment by surajkulriya** [comment]
@michaelrsweet  I am sorry for the mess I created. By the way is there still a mistake in indentation. Like I already told you its being shown differently in github, in vscodes it's looking like this:=

Screenshot from 2021-02-07 22-05-00


If you want you can close this pull request, I will make another one in which I will make all these change in a single commit to sort all this.
Thanks.

**Comment by michaelrsweet** [comment]
@surajkulriya It's OK, take your time. The final merge will be clean when it happens.

FWIW, I was hoping VS Code might be a common editor to use across platforms but I've never been able to get the tab settings to work the way I want. For Linux, GNOME's editor is OK for simple stuff, and for macOS I've been using BBEdit for years in addition to Xcode (whose editor only seems to get worse with every new release...)

**Comment by surajkulriya** [comment]
@michaelrsweet  I found [this](https://stackoverflow.com/questions/8833953/how-to-change-tab-size-on-github/23522945#23522945).
The indentation issue that's arising is because github assumes tabsize to be 8 spaces.
append `?ts=2` or `?ts=4` at the end of the [url](https://github.com/OpenPrinting/cups/pull/88/files) to change the default setting. Something like this [https://github.com/OpenPrinting/cups/pull/88/files?ts=2](https://github.com/OpenPrinting/cups/pull/88/files?ts=2).
So basically it's not an indentation issue.
Is PR mergeable now. Do tell me if there are still some more changes to make.
Thanks

**Comment by surajkulriya** [comment]
@michaelrsweet  I have made all the changes you had requested. Please do tell me if I am missing out something or if you want me to make some more changes as well.
Any other particular reason why you still have not merged this PR.
Thanks

**Comment by michaelrsweet** [comment]
@surajkulriya Sorry, I've been busy with PWG meetings and am just getting some time to look at this again. You changes look good and I'll fix any indentation problems (if any) once I've merged. Thanks!

**Comment by michaelrsweet** [comment]
[master 1feec7848] Fix indentation for cupsfilter error messages (Issue #88)

**Comment by surajkulriya** [comment]
Thanks @michaelrsweet
